core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -344,11 +362,12 @@ public static boolean getBoolean(String key) {
    * @return the value for the given key as an enum value
    */
   public static <T extends Enum<T>> T getEnum(String key, Class<T> enumType) {
-    if (!PROPERTIES.containsKey(key)) {
+    String rawValue = get(key);
```

Comment:
Could the `lookup(rawValue)` return a `null` even though it passes the check inside `get`?